ui/src/app/cron-workflows/components/cron-workflow-list/cron-workflow-list.tsx

```diff
@@ -183,3 +199,18 @@ export function CronWorkflowList({match, location, history}: RouteComponentProps
         </Page>
     );
 }
+
+function getCronNextScheduledTime(spec: CronWorkflowSpec): Date {
```

Comment:
this naming is quite confusing -- `getNextScheduledTime` already operates on cron schedules (and uses `cron-parser` directly and is imported from `shared/cron.ts`) so adding a function with `Cron` in it sounds identical.

It looks like you wrote "Cron" because it operates on the `CronWorkflowSpec` -- "Spec" would be better naming in that case.
Or "Multi" can differentiate it from the other one, since this operates on multiple schedules

```suggestion
function getSpecNextScheduledTime(spec: CronWorkflowSpec): Date {
```

Reply:
This func calculates the next scheduled time for the combination of cron expressions configured in the cron workflow spec, it handles both"schedule" and "schedules". Changing it to `getSpecNextScheduledTime` seems fine to me.

Reply:
Yea the word "Cron" was ambiguous in this case, whereas "Spec" is very specific

Reply:
Fixed in #13369 ([this file](https://github.com/argoproj/argo-workflows/pull/13369/files#diff-b7c27449b2cd699a40591fe01eb016f3532a68103e1617ac0e71f357e274d0c5))